src/ParcelRegistry.Migrator.Parcel/Infrastructure/ImportParcels.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace ParcelRegistry.Migrator.Parcel.Infrastructure
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Autofac;
+    using Be.Vlaanderen.Basisregisters.CommandHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Importer.Grb.Infrastructure;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+    using NodaTime;
+    using ParcelRegistry.Parcel;
+    using ParcelRegistry.Parcel.Commands;
+
+    internal sealed class ImportParcels
+    {
+        private readonly ILifetimeScope _lifetimeScope;
+        private readonly Dictionary<ParcelId, GrbParcel> _parcelGeometries;
+        private readonly ILogger<ImportParcels> _logger;
+        private readonly SqlStreamsTable _sqlStreamTable;
+
+        public ImportParcels(
+            ILifetimeScope lifetimeScope,
+            IConfiguration configuration,
+            Dictionary<Legacy.ParcelId, GrbParcel> parcelGeometries,
```

Comment:
Why are we passing a dictionary as parameter? We're only using its value?

Reply:
True, I just passed along what was in `Program.cs`